materialize-snowflake/sqlgen.go

```diff
@@ -304,3 +305,20 @@ FILE_FORMAT = (
 )
 COMMENT = 'Internal stage used by Estuary Flow to stage loaded & stored documents'
 ;`
+
+type tableAndFile struct {
+	Table sql.Table
+	File  string
+}
+
+// RenderTableTemplate is a simple implementation of rendering a template with a Table
+// as its context. It's here for demonstration purposes mostly. Feel free to not use it.
+func RenderTableAndFileTemplate(tf tableAndFile, tpl *template.Template) (string, error) {
+	var w strings.Builder
+	if err := tpl.Execute(&w, &tf); err != nil {
+		return "", err
+	}
+	var s = w.String()
+	log.WithField("rendered", s).WithField("tableAndFile", tf).Debug("rendered template")
```

Comment:
I'm not sure about logging this query on every transaction, even at a debug level.